repos/system_upgrade/el7toel8/libraries/vsftpdutils.py

```diff
@@ -0,0 +1,32 @@
+import errno
+import hashlib
+
+from leapp.libraries.stdlib import api
+
+VSFTPD_CONFIG_DIR = '/etc/vsftpd'
+VSFTPD_DEFAULT_CONFIG_PATH = '/etc/vsftpd/vsftpd.conf'
+STRICT_SSL_READ_EOF = 'strict_ssl_read_eof'
+TCP_WRAPPERS = 'tcp_wrappers'
+
+
+def read_file(path):
```

Comment:
```suggestion
def _read_file(path):
```

Reply:
Actually, this is public API as well :). I've documented it.